misc/python/materialize/checks/pg_cdc.py

```diff
@@ -77,6 +77,9 @@ def manipulate(self) -> List[Testdrive]:
                 true
                 """,
                 """
+                $[version>=5200] postgres-execute connection=postgres://mz_system@materialized:6877/materialize
```

Comment:
So we create `pg2` under the `materialize` user before an upgrade and after that we can no longer access it , again as a `materialize` user?

Note that this mzcompose workflow does not enable any feature flags explicitly.

Reply:
Correct. @jkosh44 Is this expected?

Reply:
What is the specific version upgrade?
If you created the table in a version before RBAC and object ownership and then upgraded to a version with RBAC and object ownership, then it would be expected. Otherwise it is not expected. 

Are you deleting and re-creating the materialize role in between upgrades?

Reply:
We don't delete and recreate the materialize role.

The entire [scenario](https://buildkite.com/materialize/nightlies/builds/2472#01885736-5fed-47ac-9653-4368429b86a9) is: `[2023-05-26T09:58:57Z] --- Testing upgrade scenario with id 31: [BeginUpgradeScenario, BeginVersion(v0.40.0), ChecksInitialize, ChecksManipulate(#1), EndVersion(v0.40.0), BeginVersion(v0.41.1), EndVersion(v0.41.1), BeginVersion(v0.42.4), EndVersion(v0.42.4), BeginVersion(v0.43.0), EndVersion(v0.43.0), BeginVersion(v0.44.3), EndVersion(v0.44.3), BeginVersion(v0.45.0), EndVersion(v0.45.0), BeginVersion(v0.46.1), EndVersion(v0.46.1), BeginVersion(v0.47.1), EndVersion(v0.47.1), BeginVersion(v0.48.4), EndVersion(v0.48.4), BeginVersion(v0.49.6), EndVersion(v0.49.6), BeginVersion(v0.50.3), EndVersion(v0.50.3), BeginVersion(v0.51.1), EndVersion(v0.51.1), BeginVersion(v0.52.4), EndVersion(v0.52.4), BeginVersion(v0.53.0), EndVersion(v0.53.0), BeginVersion(v0.54.0), ChecksManipulate(#2), EndVersion(v0.54.0), BeginVersion(v0.55.0), EndVersion(v0.55.0), BeginVersion(None), ChecksValidate, EndUpgradeScenario]`

The failure happens in `ChecksManipulate(#2)`, so in version 0.54.0, while the connection was created in `ChecksManipulate(#1)`, so in version 0.40.0. This is a pretty early version, so seems expected to me.

Reply:
@philip-stoev There might be quite a few cases like this spread across platform-matrix. Do you think we should turn off RBAC for it or fix them as we find them?